sample/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,8 @@
     <string name="hello_world">Hello World!</string>
     <string name="hello_world_2">Hello World 2!</string>
     <string name="hello_world_invisible">Hello World Invisible</string>
+
+    <!--region CompoundDrawables-->
```

Comment:
This is great! I thought it was only a C# thing. I'm definitely going to use it on my own projects as well :smile: